### PR TITLE
StandardDropdown: drop Qt 5.8 workaround

### PR DESCRIPTION
**Pull request by xiphon**

### CODE REVIEW

File: pages/settings/SettingsLayout.qml [hunk 1]
```diff
@@ -304,10 +305,6 @@ Rectangle {
             i += 1;
         }
 
-        fiatPriceProviderDropDown.update();
```
Comment (selsta) [inline pages/settings/SettingsLayout.qml:307]: Screenshot 2020-05-01 at 14 37 28

Comment (xiphon) [reply pages/settings/SettingsLayout.qml:307]: Can't reproduce. Do you see any errors in the logs? Could you try clean build?
Does the dropdown contain any options? What happens if you select one and restart the GUI?

Comment (selsta) [reply pages/settings/SettingsLayout.qml:307]: > Do you see any errors in the logs?

No

>  Could you try clean build?

Same issue

> Does the dropdown contain any options?

Yes, the dropdown looks correct

> What happens if you select one and restart the GUI?

Same issue.

Comment (selsta) [reply pages/settings/SettingsLayout.qml:307]: Note that the other dropdown menus are correct.

Comment (xiphon) [reply pages/settings/SettingsLayout.qml:284]: Fixed. Thanks for catching this.

**Pull request merged**